pinot-core/src/test/java/org/apache/pinot/core/common/RealtimeNoDictionaryTest.java

```diff
@@ -38,11 +41,13 @@
   private static final String LONG_COL_NAME = "longcol";
   private static final String FLOAT_COL_NAME = "floatcol";
   private static final String DOUBLE_COL_NAME = "doublecol";
-  private static final int NUM_ROWS = 1000;
+  private static final String STRING_COL_NAME = "stringcol";
+  private static final int NUM_ROWS = 10;
```

Comment:
Sorry this was a bad change... while I was debugging the test I added. Need to undo this

Reply:
Fixed